x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_exploration/page.tsx

```diff
@@ -74,7 +79,13 @@ export const Page: FC<{
     }
     return (
       <>
-        <AnalyticsIdSelector setAnalyticsId={setAnalyticsId} />
+        <AnalyticsIdSelectorControls setIsIdSelectorFlyoutVisible={setIsIdSelectorFlyoutVisible} />
+        {isIdSelectorFlyoutVisible ? (
```

Comment:
Could the `Edit selection` control be displayed even when there is an ID selected, as it is in the Anomaly Detection results pages? It would be a quick way of editing the selected job / model.

Reply:
Updated in 4fcc17bbbadb5fc6173e03d26d42ee94c463f130

Reply:
The 'No Analytics ID selected' label needs to be hidden when there is a job  / model selected.

![image](https://user-images.githubusercontent.com/7405507/164191428-d2770632-72e2-48d7-83c3-6e97d6bb78fa.png)

You could use an `EuiBadge` to show the selected job to be consistent with the Anomaly Explorer / SMV:

![image](https://user-images.githubusercontent.com/7405507/164191699-4029d11f-76ee-4b5d-9ee1-70ce83c2120f.png)

This would mean the job ID is shown in the page heading and in a badge for the DFA pages, but I think might look better than just showing  the `Edit job selection` link. Or could the edit link be moved into the header section?

Reply:
@peteharverson - good catch - added badge with id when selected in 3442f103587edb54900626f67926054233d1513d